docs/quickstart/csharp.md

```diff
@@ -204,7 +204,9 @@ or when you restore packages via `/path/to/nuget restore` on the command line.
 
 From the `examples/csharp/helloworld-from-cli` directory:
 
-* `/path/to/nuget install Grpc.Tools -PackagesDirectory ./packages`. 
+* mkdir packages
```

Comment:
Code snippet (three backticks)? Or at the very least make it fixed-width font (one backtick).